### PR TITLE
feat: make "Vue" a standalone category and add several alternative parsers

### PR DESCRIPTION
**Pull request by sodatea**
Currently, there are 3 different parsers available for parsing `.vue` files.

Also, the `.vue` file format differs enough from the HTML format. For example, it does not support the `doctype` declaration.

So I think it makes more sense to have a standalone "Vue" category.

### CODE REVIEW

File: website/src/parsers/vue/vue-eslint-parser.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    return key === 'children';
+  },
+
+  getNodeName(node) {
+    return node.tag;
+  },
+
+  getDefaultOptions() {
+    return {
+      sourceType: 'module'
```
Comment (IWANABETHATGUY) [inline website/src/parsers/vue/vue-eslint-parser.js]:
```suggestion
      sourceType: 'module',
```

File: website/src/parsers/vue/vue-eslint-parser.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    };
+  },
+
+  _ignoredProperties: new Set(['parent', 'tokens'])
```
Comment (IWANABETHATGUY) [inline website/src/parsers/vue/vue-eslint-parser.js]:
```suggestion
  _ignoredProperties: new Set(['parent', 'tokens']),
```

**Comment by sodatea** [comment]
I've also deployed a preview site on Netlify: https://frosty-hodgkin-d6766b.netlify.app/

**Comment by fkling** [comment]
Seems very reasonable to me, thank you!

**Pull request merged**